c++/src/kj/mutex.h

```diff
@@ -45,11 +46,24 @@ KJ_BEGIN_HEADER
 #include <pthread.h>
 #endif
 
+// There are 3 macros controlling lock tracking:
```

Comment:
Why three instead of just one that enables everything? Who would want to enable just one without the others? Is the code actually tested in every configuration?

Reply:
Technically there's only 2 macros (should update the doc to be clearer) - tracking the location the thread is blocked on & tracking the location that's holding the lock. I tested that it builds in the 4 permutations.

I figure there could be a desire to want to be able to track which mutex something is bocked on without increasing the size of the Mutex. It's probably safe to reduce it to 3 permutations by implying tracking block location if you are requesting to track the location holding the lock. That being said, since we're the only initial customer & we'll want both, I could also be convinced to just simplify further & only support 1 configuration here. Let me know.

Reply:
That brings up a good case. How do I make sure the new config is covered in CI to avoid accidental bitrot before a roll-deps lands this in EW?

Reply:
I think I figured out how to add coverage. Was conservative for now in case you were concerned about build time but I can add coverage for the mix'n'match combination of options as well.